vignettes/FDZ_Data_Cleaning.Rmd

```diff
@@ -14,8 +14,8 @@ For this purpose, operations such as recoding of identifier variables, emptying
 
 - Removing all non-ASCII characters
 - Appropriate missing tags
-- Non-informative identifier variables
-- Transforming strings to labeled numeric variables
+- Adding non-informative identifier variables
```

Comment:
LGTM!
Should "appropriate missings tags" be changed to "Setting appropriate missing tags" or sth similar as well?